packages/flat-components/src/components/ClassroomPage/Countdown/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef, useState, useEffect, useCallback } from "react";
+import classnames from "classnames";
+import { useTranslation } from "react-i18next";
+import "./style.less";
+
+export type CountdownProps = {
+    state: "paused" | "started";
+    beginTime: number;
+};
+
+function renderTime(seconds: number): string {
+    const h = Math.floor(seconds / 3600);
+    const m = Math.floor((seconds % 3600) / 60);
+    const s = Math.floor((seconds % 3600) % 60);
+    return (
+        (h > 0 ? String(h).padStart(2, "0") + ":" : "") +
+        String(m).padStart(2, "0") +
+        ":" +
+        String(s).padStart(2, "0")
+    );
+}
```

Comment:
use data-fns function

```suggestion
const prefixInteger = (number: number): string => {
    return String(number).padStart(2, "0");
}

const renderTime = (seconds: number): string => {
    const { hours, minutes, seconds } = intervalToDuration({start: 0, end: seconds * 1000});

    return `${prefixInteger(hours)}:${prefixInteger(minutes)}:${prefixInteger(seconds)}`
}
```